README.md

```diff
@@ -1,72 +1,84 @@
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/unwrapped/assets/7949047/b8a01018-e5dd-4d0f-a5da-ab3cfe039862" width="100%" /></a>
+
+[![Unwrapped Logo](https://github.com/middlewarehq/unwrapped/assets/7949047/b8a01018-e5dd-4d0f-a5da-ab3cfe039862)](https://unwrapped.dev)
 
 <blockquote align="center">
   <h3>
     <i>
-      Spotify Unwrapped - for devs
+      Dive into Unwrapped - for coders
       <br/>
-      Showcase your contributions just like you flex your music!
+      Reveal your coding journey just as you showcase your musical taste!
       <br />
-      Check it out at https://unwrapped.dev
+      Explore more at  https://unwrapped.dev
```

Comment:
I think this change isn't needed for now.
Happy to hear your reasoning behind why it helps, if it does.